polkadot-parachains/integritee-runtime/src/lib.rs

```diff
@@ -264,7 +333,115 @@ impl pallet_sudo::Config for Runtime {
 }
 
 parameter_types! {
-	pub const MinVestedTransfer: Balance = 100 * TEER;
```

Comment:
That's a little confusing diff - MinVestedTransfer was not removed, but changed to match the Solochain to
`pub const MinVestedTransfer: Balance = 1 * TEER;` (see https://github.com/integritee-network/parachain/pull/83/files#diff-3f0515d808b2b888ea9793f1334fbd7609bcb649a9f9601042c13ee3b6054afbR444)

Reply:
Should stay 100 for now